### DISCUSSION TITLE
When can I use copilot chat of IDEA ?

### DISCUSSION THREAD

**Discussion by imwule** [community/community#72910] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I like using IDEA.

**Comment by isharax9** [comment, marked as answer]
There is no official release date for Copilot Chat for IntelliJ IDEA yet. However, in a recent I heard GitHub discussion, a GitHub Copilot representative stated that they are working on supporting IntelliJ IDEA and other IDEs. They did not provide an ETA for when this support will be available, but they stated that they will update the community when they have more details to share.

In the meantime, there is a third-party plugin called ChatGPT Copilot which can be used to add Copilot Chat functionality to IntelliJ IDEA. This plugin is still under development, but it has received positive reviews from users.

It seems currently only a vscode version exists: [https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat)

I hope this information is helpful. Please mark this as answer if it helps.

**Reply by imwule** [reply]
> There is no official release date for Copilot Chat for IntelliJ IDEA yet. However, in a recent I heard GitHub discussion, a GitHub Copilot representative stated that they are working on supporting IntelliJ IDEA and other IDEs. They did not provide an ETA for when this support will be available, but they stated that they will update the community when they have more details to share.
> 
> In the meantime, there is a third-party plugin called ChatGPT Copilot which can be used to add Copilot Chat functionality to IntelliJ IDEA. This plugin is still under development, but it has received positive reviews from users.
> 
> It seems currently only a vscode version exists: [https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat)
> 
> I hope this information is helpful. Please mark this as answer if it helps. 

Thank you.
